04_setup_ironic.sh

```diff
@@ -33,26 +33,59 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
         [ -e "$REPOPATH" ] || git clone $IMAGE $REPOPATH
         cd $REPOPATH
         export $IMAGE_VAR=${IMAGE##*/}:latest
-        export $IMAGE_VAR=$LOCAL_REGISTRY_ADDRESS/localimages/${!IMAGE_VAR}
-        sudo podman build -t ${!IMAGE_VAR} .
+        export $IMAGE_VAR=$LOCAL_REGISTRY_ADDRESS:$LOCAL_REGISTRY_PORT/localimages/${!IMAGE_VAR}
+        sudo podman build --authfile $REGISTRY_CREDS -t ${!IMAGE_VAR} .
         cd -
-        sudo podman push --tls-verify=false ${!IMAGE_VAR} ${!IMAGE_VAR}
+        sudo podman push --tls-verify=false --authfile $REGISTRY_CREDS ${!IMAGE_VAR} ${!IMAGE_VAR}
     fi
 
     # Update the bootstrap and master nodes to treat LOCAL_REGISTRY_ADDRESS as insecure
     mkdir -p $OPENSHIFT_INSTALL_PATH/data/data/bootstrap/baremetal/files/etc/containers
-    echo -e "[registries.insecure]\nregistries = ['${LOCAL_REGISTRY_ADDRESS}']" > $OPENSHIFT_INSTALL_PATH/data/data/bootstrap/baremetal/files/etc/containers/registries.conf
+    echo -e "[registries.insecure]\nregistries = ['${LOCAL_REGISTRY_ADDRESS}:${LOCAL_REGISTRY_PORT}']" > $OPENSHIFT_INSTALL_PATH/data/data/bootstrap/baremetal/files/etc/containers/registries.conf
     cp assets/templates/99_local-registry.yaml.optional assets/templates/99_local-registry.yaml
```

Comment:
We can probably remove this now as your using "imageContentSources".

Reply:
Thanks @derekhiggins , if U could help testing this it would be much appreciated

Reply:
We set _imageContentSources_ just when _MIRROR_IMAGES_ is set to _true_, so iiuc it's still needed